### PR TITLE
docs: add ppas and fix typos

### PR DESCRIPTION
**Pull request by orndorffgrant**
## Proposed Commit Message

> docs: small readme fixes
>
> * Fixes some small grammatical mistakes.
> * Adds details of the three PPAs available for the client.
> * Adds the `make testdeps` step to the Testing section with a brief explanation.

## Test Steps

Read the proposed changes to the readme and decide if they make sense or not.

## Desired commit type::

 - [x] Squash and merge: Using the "Proposed Commit Message"
 - [ ] Create a merge commit and use "Proposed Commit Message"
 - [ ] Rebase and merge, no merge commit, rely only on existing commit messages

## Checklist:

 - [ ] I have updated or added any unit tests accordingly
 - [ ] I have updated or added any integration tests accordingly
 - [x] I have updated or added any documentation accordingly

### CODE REVIEW

File: README.md
Comment (orndorffgrant) [inline README.md:31]: First of all: is this the gist of what we want to add for these and is it in an appropriate place?

I made a best guess for these descriptions based on the names. Please let me know if anything is wrong.

File: README.md
Comment (orndorffgrant) [inline README.md:132]: This is my attempt at a short description of why we have `tox-pip-version` based on my memory of what @lucasmoura was saying yesterday. Please chime in if its incorrect/too long/unnecessary/etc.

File: README.md
Comment (orndorffgrant) [inline README.md:203]: The letters here weren't rendering properly. I prefer the look of the letters in plaintext, but when rendered as html, the sub-ordered list looks better to me. LMK if you'd rather I change it back

File: README.md
Comment (lucasmoura) [reply README.md:132]: I think this is a great description of the problem. Thanks @orndorffgrant

**Review by lucasmoura — APPROVED** [review]

**Review by lucasmoura — CHANGES_REQUESTED** [review]
@orndorffgrant I think we can also add the `make testdeps` step before running the uaclient tests.
After adding that, I can merge this PR

**Comment by orndorffgrant** [comment]
Thanks @lucasmoura! the `make testdeps` step is there underneath the comment about tox-pip-version: https://github.com/canonical/ubuntu-advantage-client/pull/1496/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R137

My comment above probably distracted from it - is it in the right place?

**Review by lucasmoura — APPROVED** [review]
You are totally right here @orndorffgrant. My mistake here

**Comment by lucasmoura** [comment]
This PR is touching only README files, so I will not wait for the CI run on it

**Pull request merged**
